File: mongodb/python/scripts/insert_one.py
from pymongo import MongoClient
from db import client
 
# get the db to use
dbname=client['course']

# use or create a collection
# dbname.billings
billingCollection=dbname['billings']   
# document to insert
bill1={
    "code_billing": 1,
    "description": "This is an example",
    "total": 100,
    "products": ["tomato","pear","lemons"]
}

# Insert document
result=billingCollection.insert_one(bill1)

# print the insert document
print(result)

# get the collections of db
collections=dbname.list_collection_names()

# print the collection list
print(collections)
for collection in collections:
    print(collection)

File: mongodb/nodejs/insert_one.js
// import mongo client
const {client} = require('./db');


const main = async ()=> {
    try {
        //open connection with mongo
        await client.connect();
        console.log('Connection successfully to ATLAS');
        // get db
        const dbName = client.db('course');
        // get ventas collection
        const collection = dbName.collection('ventas');
        // create venta obj
        const sell1 = {
            cod_sell : 100,
            name: 'Cliente 1',
            address: 'Calle falsa 123',
            billDate: new Date(),
            total: 200
        }
        // insert document
        let result = await collection.insertOne(sell1);
        console.log(result.insertedId)
    } catch (error) {
        console.log("**__** ~ main ~ error:", error);        
    } finally {
        // close connection
        await client.close()
    }
}

main();